src/controllers/__tests__/item.test.js

```diff
@@ -393,14 +439,14 @@ describe('Items Endpoints', () => {
     });
 
     test('Should throw an error if there is no item with the specified ID', async () => {
-      const response = await request(server).delete(
-        '/api/items/6208e47a5fe21cc475419234'
-      );
+      const response = await request(server)
+        .delete('/api/items/6208e47a5fe21cc475419234')
```

Comment:
Let's ensure we don't use any hardcoded IDs in the requests because for later maintenance it can get difficult to locate these. Similar to the previous requests, let's use a const like `itemId` and reference it here.

Reply:
Ok I will add a constant like `wrongItemId`